qa/L0_stability_metrics/check_results.py

```diff
@@ -49,6 +48,7 @@ def check_metrics_stability(self):
         pathname = os.path.join(self._result_path, 'result_*.csv')
         csv_contents = []
         for filename in glob.glob(pathname):
+            print('Opening file <' + str(filename) + '>')
```

Comment:
Did you want to leave these print statements in?

Reply:
Yes - when I ran it, and all tests passed, I did not see any confirmation that anything ran at all.  There was a part of me that thought maybe no test was performed since nothing appeared.  So, this print() and the other print() are there to confirm that files were opened and tests were performed.